### PR TITLE
add a line by Eric

### PR DESCRIPTION
**Pull request by Eric-bs**
nothing more

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # hello_git
+
 add a line
+add by ldp
```
Comment (ldp12416) [inline README.md]: why keep the old line?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
 # hello_git
+
 add a line
```
Comment (ldp12416) [inline README.md]: remove this line

Comment (Eric-bs) [reply README.md]: OK

File: README.md (hunk 1 above)
Comment (Eric-bs) [reply README.md]: OK

**Pull request merged**